aws/resource_aws_db_cluster_snapshot_test.go

```diff
@@ -39,6 +40,8 @@ func TestAccAWSDBClusterSnapshot_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "status", "available"),
 					resource.TestCheckResourceAttr(resourceName, "storage_encrypted", "false"),
 					resource.TestMatchResourceAttr(resourceName, "vpc_id", regexp.MustCompile(`^vpc-.+`)),
+					testAccCheckDbClusterSnapshotExists("aws_db_cluster_snapshot.test", &v),
```

Comment:
This check appears to be a duplicate of the existing one on line 28.

Reply:
I removed duplicated line.